.travis.yml

```diff
@@ -16,3 +17,12 @@ script:
   - sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck scalafmtSbtCheck test:scalafmtCheck clean coverage test
 after_success:
   - sbt coverageReport coveralls
+before_deploy:
+  - sbt docker:publishLocal
+  - echo "$DOCKER_TOKEN" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  - docker tag "open360/api" "$DOCKER_IMAGE_NAME:$([ "$TRAVIS_TAG" == "" ] && echo "latest" || echo "$TRAVIS_TAG")"
```

Comment:
Hot about appending hash commit instead of `latest` for non-tagged commits?

Reply:
It will cause docker repository to be overflown with bunch of images, what really the upside of such approach?

Reply:
let's do it for master only and for branches as image:branch

Reply:
@yaroslavdudchenko The upside is that you have a built image for each of your commits. That makes it easier for you to deploy earlier versions of the application

Reply:
@maximskripnik yes, I can get behind your idea, but in closed repo and when you are really in need of constant version swapping. In this particular case it will not be of much use.